docs/testcases.html

```diff
@@ -98,8 +98,14 @@ <h2 id="purpose">Placeholder Index</h2>
                             <a href="testcases/TC06-006-pass.html">TC06-006-pass</a>
 						  </li>
 						  <li>
-                            <a href="testcases/TC06-007-pass.html">TC06-007-pass</a>
-                          </li>
+                            <a href="testcases/TC06-007-fail.html">TC06-007-fail</a>
+						  </li>
+						  <li>
+                            <a href="testcases/TC06-008-fail.html">TC06-008-fail</a>
+						  </li>
+						  <li>
+                            <a href="testcases/TC06-009-fail.html">TC06-009-fail</a>
+						  </li>
                         </ul>
                     </li>
                     <li>10. Forms
```

Comment:
testcases.html:

- Recommend holding off on these changes to expose access to these test cases until code updates are complete on them.

Reply:
OK, removed.